src/02 - Object types.md

```diff
@@ -405,6 +405,8 @@ The following notation is used in the tables below:
 | M013 | number          | journalaar                    |    x     |      x      |           |  |  cw  |  x   |   x   |  x   |               |                                                                                                                                                                                                                                                                                                        |                                                                                                                                                        |
 | M014 | number          | journalsekvensnummer          |    x     |      x      |           |  |  cw  |  x   |   x   |  x   |               |                                                                                                                                                                                                                                                                                                        |                                                                                                                                                        |
 | M015 | number          | journalpostnummer             |    x     |      x      |           |  |  cw  |  x   |   x   |  x   |               |                                                                                                                                                                                                                                                                                                        |                                                                                                                                                        |
+|      | string          | journalansvarlig              |    x     |      x      |           |  |  x   |  x   |   x   |  x   |               | <p>An identifier of the person responsible for this Journalpost (similar to M307 Saksmappe#saksbehandler)</p><p>&bull; may not be supported by all systems or partially supported</p>                                                                                                                  |                                                                                                                                                        |
```

Comment:
`journalansvarlig` contains the name of the person, similarly to `saksansvarlig`. I think this is not reflected in the documentation correctly. We need the comment to state that `journalansvarlig` contains a person's name, while `journalansvarligBrukerIdent` contains the corresponding user ID.

Reply:
Addressed. Please take a look at the change.